package.json

```diff
@@ -72,11 +72,13 @@
     "echarts-for-react": "^3.0.2",
     "eslint-mdx": "^3.1.5",
     "events": "^3.3.0",
-    "hi-profiles": "^1.0.6",
     "glob": "^10.3.15",
+    "hi-profiles": "^1.0.6",
+    "i": "^0.3.7",
     "i18next": "^23.11.4",
     "i18next-browser-languagedetector": "^7.2.1",
     "lodash-es": "^4.17.21",
+    "npm": "^10.8.1",
```

Comment:
Are these packages relevant? If they were added by mistake, remove them and update the package-lock file too

Reply:
i think i typed `npm i` twice for installing package 😅
will remove them